client/src/pages/platform/workflow-editor/components/node-details-tabs/OutputTabSampleDataDialog.tsx

```diff
@@ -8,34 +8,86 @@ import {
     DialogTitle,
 } from '@/components/ui/dialog';
 import Editor from '@monaco-editor/react';
-import {useState} from 'react';
-
-const sampleOutput = {
-    country: 'USA',
-    people: [
-        {
-            age: 28,
-            firstName: 'Joe',
-            gender: 'male',
-            lastName: 'Jackson',
-            number: '7349282382',
-        },
-    ],
-};
+import {editor} from 'monaco-editor';
+import {useEffect, useState} from 'react';
+import IStandaloneCodeEditor = editor.IStandaloneCodeEditor;
+
+const placeholder = (
+    <>
+        <div>{'//'}Write sample output value, for example:</div>
+        <div>{'{'}</div>
+        <div className="pl-4">{'"country": "USA"'}</div>
+        <div className="pl-4">{'"people": ['}</div>
+        <div className="pl-8">{'{'}</div>
+        <div className="pl-12">{'"firstName": Joe'}</div>
+        <div className="pl-12">{'"lastName": Jackson'}</div>
+        <div className="pl-12">{'"gender": Male'}</div>
+        <div className="pl-12">{'"age": 28'}</div>
+        <div className="pl-12">{'"number": 7349282382'}</div>
+        <div className="pl-8">{'}'}</div>
+        <div className="pl-4">{']'}</div>
+        <div>{'}'}</div>
+    </>
+);
```

Comment:
Jel ovo baš najelegantniji način za postić šta želiš?

Postoji `<pre>` HTML element koji bi moga bit bolji za ovo